src/core/packet_builder.c

```diff
@@ -828,13 +828,14 @@ QuicPacketBuilderFinalize(
         //
         QUIC_PACKET_SPACE* PacketSpace = Connection->Packets[Builder->EncryptLevel];
         PacketSpace->CurrentKeyPhaseBytesSent += (PayloadLength - Builder->EncryptionOverhead);
+        uint16_t MaxMtu = QuicConnGetMaxMtuForPath(Connection, Builder->Path);
 
         //
         // Check if the next packet sent will exceed the limit of bytes per
         // key phase, and update the keys. Only for 1-RTT keys.
         //
         if (Builder->PacketType == SEND_PACKET_SHORT_HEADER_TYPE &&
-            PacketSpace->CurrentKeyPhaseBytesSent + CXPLAT_MAX_MTU >=
+            PacketSpace->CurrentKeyPhaseBytesSent + MaxMtu >=
```

Comment:
I'm thinking this change is unnecessary. We just need a rough value here. I think just using a (any) #define is best.

Reply:
Ok. I can just change it to CXPLAT_MAX_MTU.